source/guide_gitea.rst

```diff
@@ -96,93 +157,276 @@ and a new config file at ``~/gitea/custom/conf/app.ini`` to specify the port and
+* Naming the repositories path.
+* Set the eye-friendly dark theme as default theme, set mailing, set time and set logging options.
+
+For more informations about the possibilities and configuration options see the Gitea documentation_ and the Gitea `configuration sample <https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample>`_.
 
-Configure web server
---------------------
+.. note:: The minimum improvements are written in the sections of ``[database]``, ``[security]`` and ``[service]``.
 
-.. note::
+.. warning:: Replace ``isabell`` with your username, fill the database password ``PASSWD =`` with yours and enter the generated random into ``SECRET_KEY =``.
 
-    Gitea is running on port 9000.
+.. code-block:: ini
 
-.. include:: includes/web-backend.rst
+  APP_NAME = Gitea
+  RUN_USER = isabell
+  RUN_MODE = prod ; Either "dev", "prod" or "test", default is "dev".
+  
+  [server]
+  HTTP_PORT            = 9000
+  DOMAIN               = isabell.uber.space
+  ROOT_URL             = https://%(DOMAIN)s
+  STATIC_URL_PREFIX    = 
+  OFFLINE_MODE         = true
+  DISABLE_SSH          = false
+  SSH_DOMAIN           = isabell.uber.space
+  SSH_PORT             = 22
+  LANDING_PAGE         = explore ; [home, explore, organizations, login]
+  LFS_START_SERVER     = true
+  LFS_CONTENT_PATH     = /home/isabell/gitea/data/lfs
+  LFS_JWT_SECRET       = 
+  LFS_HTTP_AUTH_EXPIRY = 24m
+  
+  [database]
+  DB_TYPE           = mysql
+  HOST              = 127.0.0.1:3306
+  NAME              = isabell_gitea
+  USER              = isabell
+  PASSWD            = <MySQL_PASSWORD>
+  SSL_MODE          = disable
+  
+  [security]
+  INSTALL_LOCK          = true
+  MIN_PASSWORD_LENGTH   = 8
+  PASSWORD_COMPLEXITY   = lower,digit
+  PASSWORD_HASH_ALGO    = pbkdf2 ; [pbkdf2, argon2, scrypt, bcrypt]
+  LOGIN_REMEMBER_DAYS   = 5
+  CSRF_COOKIE_HTTP_ONLY = true
+  INTERNAL_TOKEN        = 
+  SECRET_KEY            = <RANDOM_32_CHARS>
+  
+  [service]
+  DISABLE_REGISTRATION                   = true
+  SHOW_REGISTRATION_BUTTON               = false
+  REQUIRE_EXTERNAL_REGISTRATION_PASSWORD = false
+  ALLOW_ONLY_EXTERNAL_REGISTRATION       = false
+  REQUIRE_SIGNIN_VIEW                    = true
+  ENABLE_CAPTCHA                         = false
+  REGISTER_EMAIL_CONFIRM                 = true
+  ACTIVE_CODE_LIVE_MINUTES               = 60
+  RESET_PASSWD_CODE_LIVE_MINUTES         = 60
+  ENABLE_NOTIFY_MAIL                     = false
+  SHOW_MILESTONES_DASHBOARD_PAGE         = true
+  DEFAULT_ALLOW_CREATE_ORGANIZATION      = true
+  DEFAULT_ORG_VISIBILITY                 = private ; [public, limited, private]
+  DEFAULT_ORG_MEMBER_VISIBLE             = false
+  DEFAULT_ENABLE_TIMETRACKING            = true
+  DEFAULT_KEEP_EMAIL_PRIVATE             = true
+  NO_REPLY_ADDRESS                       = noreply.isabell.uber.space
+  
+  [repository]
+  ROOT = /home/isabell/gitea/repositories
```

Comment:
The default path ` ~/gitea-repositories` seems fine to me and makes the new guide more compatible to installations based on the old one.

Reply:
Done.